students/ian_letourneau/Lesson07/html_render.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 5/30/2018
+
+from io import StringIO
+
+"""A script to define classes to later be initialized by the
+run_html_render.py script."""
+
+
+class Element:
+    """Parent class for all html elements to be created."""
+
+    def __init__(self, open_tag="<html>",
+                 close_tag="</html>", content=None, **kwargs):
```

Comment:
it is better to define tag as class variable:
```
class Element:
    tag = "html" 

    def __init__(...):
        ....
```
then your subclass become very simple (this is all that would need to be defined):
```
class P(Element):
    tag = "p"
```

the rest of the logic like open / close tags etc should be in the function that is using it (`Element`/`render`)